packages/altive_lints/lib/src/lints/avoid_hardcoded_japanese.dart

```diff
@@ -40,6 +40,9 @@ class AvoidHardcodedJapanese extends DartLintRule {
     ErrorReporter reporter,
     CustomLintContext context,
   ) {
+    if (resolver.source.uri.pathSegments.contains('/test/')) {
+      return;
+    }
```

Comment:
![badge](https://img.shields.io/badge/review-imo-important)
From reading the documentation, isn't `/` unnecessary?
Read the documentation comments when implementing.

Reply:
It seems unnecessary since pathSegments are stored in a list with each slash.
We will remove the slashes in an additional commit.